docs/source/topics/dns-service.rst

```diff
@@ -0,0 +1,21 @@
+===========
+DNS Service
+===========
+
```

Comment:
May be a tutroial for writing in memory cache like: https://github.com/sibiryakov/frontera-google/blob/master/bc/broadcrawl/__init__.py#L14 and how it can be used with the crawling strategy will be really helpful. I think adding https://github.com/sibiryakov/frontera-google to examples will be really helpful for new comers.